third-party/chpl-venv/chpldoc-sphinx-project/source/conf.py

```diff
@@ -53,7 +53,10 @@
 
 author_text = os.environ.get('CHPLDOC_AUTHOR', 'AUTHOR TEXT')
 
-copyright = u'2015, {0}'.format(author_text)
+if len(author_text):
+    copyright = u'2015, {0}'.format(author_text)
+else:
+    copyright = u'2015'
```

Comment:
Except maybe we should be smarter about choosing a copyright year (but that'd be good to handle in another PR, probably)

Reply:
I actually don't know about the stuff like how to choose a copyright year and  what it means... So I just left it there.